src/main/java/org/opensearch/security/dlic/rest/api/InternalUsersApiAction.java

```diff
@@ -13,20 +13,20 @@
  *  permissions and limitations under the License.
  */
 
-package com.amazon.opendistroforelasticsearch.security.dlic.rest.api;
-
-import com.amazon.opendistroforelasticsearch.security.DefaultObjectMapper;
-import com.amazon.opendistroforelasticsearch.security.auditlog.AuditLog;
-import com.amazon.opendistroforelasticsearch.security.configuration.AdminDNs;
-import com.amazon.opendistroforelasticsearch.security.configuration.ConfigurationRepository;
-import com.amazon.opendistroforelasticsearch.security.dlic.rest.validation.AbstractConfigurationValidator;
-import com.amazon.opendistroforelasticsearch.security.dlic.rest.validation.InternalUsersValidator;
-import com.amazon.opendistroforelasticsearch.security.privileges.PrivilegesEvaluator;
-import com.amazon.opendistroforelasticsearch.security.securityconf.Hashed;
-import com.amazon.opendistroforelasticsearch.security.securityconf.impl.CType;
-import com.amazon.opendistroforelasticsearch.security.securityconf.impl.SecurityDynamicConfiguration;
-import com.amazon.opendistroforelasticsearch.security.ssl.transport.PrincipalExtractor;
-import com.amazon.opendistroforelasticsearch.security.support.SecurityJsonNode;
+package org.opensearch.security.dlic.rest.api;
```

Comment:
Why some packages in `dlic` changes and some are not?

Reply:
Considering the third-party dependencies such as ldap, the `com.amazon.dlic` is not changed. Only changed the package of security plugin `com.amazon.opendistroforelasticsearch.security`